circuit-lib/circuit-lib.circom/scripts/checkBuildCircuit.sh

```diff
@@ -45,4 +45,8 @@ check_file_modified ../../system-programs/programs/verifier_program_one/src/veri
 
 check_file_modified ../../zk.js/build-circuits/transactionMasp2Main.zkey
 check_file_modified ../../zk.js/build-circuits/transactionMasp10Main.zkey
-check_file_modified ../../zk.js/build-circuits/transactionApp4Main.zkey
\ No newline at end of file
+check_file_modified ../../zk.js/build-circuits/transactionApp4Main.zkey
+
+check_file_modified ../../zk.js/build-circuits/transactionMasp2Main_js/transactionMasp2Main.wasm
+check_file_modified ../../zk.js/build-circuits/transactionMasp10Main_js/transactionMasp10Main.wasm
+check_file_modified ../../zk.js/build-circuits/transactionApp4Main_js/transactionApp4Main.wasm
```

Comment:
please revert changes in this file it's not related to your pr

Reply:
Previously, it was the old `light-zk.js` before #270 -> so i needed to adapt it to make it work